src/app/dim-api/dim-api-helper.ts

```diff
@@ -99,12 +99,13 @@ export async function authenticatedApi<T>(config: HttpClientConfig): Promise<T>
     return response.json();
   }
 
+  let responseData: { error: string; message: string } | undefined;
   try {
-    const responseData = await response.json();
-    if (responseData.error) {
-      throw new Error(`${responseData.error}: ${responseData.message}`);
-    }
+    responseData = await response.json();
   } catch {}
+  if (responseData?.error) {
```

Comment:
This was just me realizing that we were swallowing the error we tried to throw from DIM API.